src/PodiumdAdapter.Web/Endpoints/ContactmomentenClientConfig.cs

```diff
@@ -354,61 +345,66 @@ private static void HandleAfdelingGroepMedewerker(JsonNode? json, bool isContact
         {
             if (json == null) return;
             var actor = json["actor"];
-            var soortActor = actor?["soortActor"]?.GetValue<string>();
 
-            // als het geen contactverzoek betreft,
-            // dan wordt de afdeling/groep van het contactmoment doorgestuurd als afdeling/groep van het contact
+            // als het een contactmoment betreft,
+            // dan wordt de afdeling van het contactmoment doorgestuurd als afdeling van het contact 
+            // als het een contactverzoek betreft dan doen we niks met de afdeling/groep van het contactmoment
             if (!isContactverzoek)
             {
-                AfdelingOfGroepOvernemenVanHetContactmoment(json);
-                return;
-            }
-
-            // Als het een contactverzoek betreft en er is een afdeling/groep gekozen bij het contactverzoek,
-            // .. dan wordt die afdeling/groep doorgestuurd als afdeling/groep van het contact
-            // .. dan doen we niks met de afdeling/groep van het contactmoment
-            if (TryGetOrganisatorischeEenheid(actor, out var naamVanAfdelingOfGroep, out var propertyNaamVoorAfdelingOfGroep))
-            {
-                json[propertyNaamVoorAfdelingOfGroep] = naamVanAfdelingOfGroep;
+                AfdelingOvernemenVanHetContactmoment(json);
                 return;
             }
+           
+            var isContactverzoekVoorMedewerker = IsContactverzoekVoorMedewerker(actor);
 
             // Als het een contactverzoek betreft en er is een medewerker gekozen bij het contactverzoek,
-            // .. dan wordt die medewerker doorgestuurd als de behandelaar van het contact
-            // .. dan doen we niks met de afdeling/groep van het contactmoment
-            if (TryGetActorUserName(actor, out var username))
+            // .. dan wordt die medewerker doorgestuurd als de behandelaar van het contact            
+            if (isContactverzoekVoorMedewerker)
             {
+                var username = actor?["identificatie"]?.GetValue<string>() ?? "";
                 var behandelaar = json.GetOrSetProperty("behandelaar", () => new JsonObject());
                 behandelaar["gebruikersnaam"] = username;
             }
+
+            // Als het een contactverzoek betreft en er is een afdeling/groep gekozen bij het contactverzoek,
+            // .. dan wordt die afdeling/groep doorgestuurd als afdeling/groep van het contact
+            if (TryGetOrganisatorischeEenheid(actor, isContactverzoekVoorMedewerker, out var value, out var propertyName))
```

Comment:
ik zie wel dat je de informatie isContactverzoekVoorMedewerker nodig hebt in deze functie, maar dat wil niet zeggen dat het het logisch is om deze functie dat te laten bepalen en dat in een out parameter te stoppen.